docs/README.md

```diff
@@ -24,6 +24,8 @@ yarn build
 
 This command generates static content into the `build` directory and can be served using any static contents hosting service.
 
+**Note**: You can use `npm run serve` to test your build locally.
```

Comment:
```suggestion
**Note**: You can use `yarn serve` to test your build locally.
```

Reply:
@codesankalp could you pls explain why I should replace `npm run serve` with `yarn serve` ? I don't see much difference in it.

Reply:
@ashwinijha6 You have added the same note line twice. As the first few lines of the README include `yarn start`, `yarn build` etc.. then according to the flow there should be `yarn serve` for testing the build locally.
I think it is a valid reason.

Reply:
Yes that was on purpose @codesankalp. I can do one thing that is change the command after `yarn build` to `yarn serve` & let the second command after `npm run build` be as it is OR remove the second command. What do you think?